Tests/WebTest/SitemapTest.php

```diff
@@ -87,14 +90,19 @@ public function getFormats()
                          <xhtml:link href="http://localhost/sitemap-aware?_locale=de" hreflang="de" rel="alternate"/>
                      </url>
                      <url>
+                         <loc>http://localhost/sitemap-aware-last-mod-date</loc>
+                         <lastmod>2016-07-06T00:00:00+02:00</lastmod>
```

Comment:
I'm not *that* old! I was more refering to where the actual data is set as I couldn'T figure out at first where that happens. The referenced line is the point where you set the comparison date but the question is more where the value is set that you are comparing against. Whatever: I think I found it and it get's fed from the database. But where is that database-value set in the first place? As that's where the offset-information is stored.

As the question is **when**  the value was changed and you can't reliably check **where** (geographically) it was changed, I'd always store the value in UTC and execute all the tests against it using ```new DateTimezone('UTC')``` for checks. I'D also check whether the dates are always written as UTC to the database.